### PR TITLE
LPS-129013 Adding color back to google drive icons

### PR DESCRIPTION
**Pull request by boton**
Hi team,

We need to add data attributes in the management toolbar dropdowns items again because we lose the Google icon's colors.

**Test plan**

1. Go to System Settings -> Documents and Media -> Google Drive.
2. Set some random values for the key and secret fields and save.
3. Go to Documents And Media.
4. Click on the plus button.

**Before the fix**
image

**After the fix**
image

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

**Comment by julien** [comment]
Changes look reasonable to me, thanks @boton and @4lejandrito 
Going to test this before confirming my "approval".

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js [hunk 1]
```diff
@@ -102,6 +103,7 @@ const CreationMenu = ({
 					onCreationMenuItemClick(event, {item});
 				}}
 				symbolLeft={item.icon}
+				{...getDataAttributes(item.data)}
```
Comment (jbalsas) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js:106]: Interesting!! 👍

**Comment by julien** [comment]
@boton this works and I think it's OK. 

However, I also think that instead of setting `data-*` attributes in "camelCase", we should instead set them in lowercase like it's supposed to be done. If you read [this article](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/data-*) on `data-*` attributes, you'll see 

> The * may be replaced by any name following the production rule of XML names with the following restrictions:

> The name must not start with xml (case-insensitive).
> The name must not contain any colon characters (:).
> The name must not contain any capital letters.

**Comment by boton** [comment]
Yep @julien I think that we have some data-attributes in CamelCase across the site but not, in this case, https://github.com/liferay/liferay-portal/blob/aa7b7141b63ef4781020c31783b3cdd13fc883f0/modules/apps/document-library-opener/document-library-opener-google-drive-web/src/main/java/com/liferay/document/library/opener/google/drive/web/internal/portlet/toolbar/contributor/DLOpenerGoogleDriveDLPortletToolbarContributorContext.java#L113 :D

**Comment by julien** [comment]
@boton forget my last comment : we were actually "loosing" the data attributes. :+1: So all good for me.

**Comment by julien** [comment]
ci:forward

**Pull request closed without merging**